tests/Behat/RdfEntityContext.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines step definitions specifically for testing the RDF entities.
+ *
+ * This is needed due to issues with the Kernel testing of the RDF entity module
+ * in Drupal 8.6.
+ */
+class RdfEntityContext extends RawDrupalContext {
+
+  /**
+   * Before scenario hook for the rdf entities tests.
+   *
+   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
+   *   The Hook scope.
+   *
+   * @BeforeScenario @rdf-test
+   */
+  public function beforeScenarioRdfSetup(BeforeScenarioScope $scope): void {
+    // In case a URI is being set in the scenario, keep track of the old one.
+    $existing = \Drupal::configFactory()->get('oe_content.settings')->get('provenance_uri');
+    \Drupal::state()->set('behat_rdf_test_existing_provenance', $existing);
+  }
+
+  /**
+   * After scenario hook for the rdf entities tests.
+   *
+   * @param \Behat\Behat\Hook\Scope\AfterScenarioScope $scope
+   *   The Hook scope.
+   *
+   * @AfterScenario @rdf-test
+   */
+  public function afterScenarioRdfCleanUp(AfterScenarioScope $scope): void {
+    // If a provenance URI was changed, restore the previous one..
+    $provenance_uri = \Drupal::state()->get('behat_rdf_test_existing_provenance');
+    \Drupal::configFactory()
+      ->getEditable('oe_content.settings')
+      ->set('provenance_uri', $provenance_uri)
+      ->save();
+    \Drupal::state()->delete('behat_rdf_test_existing_provenance');
+
+    $label = \Drupal::state()->get('behat_rdf_test_created_entity');
```

Comment:
Take a look at the [implementation in Joinup](https://github.com/ec-europa/joinup-dev/blob/develop/web/modules/custom/solution/solution.behat.inc), we can take some [stuff from there](https://github.com/ec-europa/joinup-dev/blob/develop/tests/src/Traits/RdfEntityTrait.php).